apps/app/src/app/account/account-home/account-home.component.html

```diff
@@ -20,16 +20,42 @@ <h2>{{ 'account.characters' | translate }}</h2>
 
     <div fxFlex="0 0 auto"
          class="left-column">
-      <div class="container switch-action"><a [routerLink]="'../' + (kbSlug | async)">
-          <span [innerHtml]="'account.back' | translate"></span>
-          <svg-icon src="assets/icons/rounded_chevron_right.svg"></svg-icon>
-        </a>
-      </div>
-
       <div class="container">
         <span class="totals"
               [innerHtml]="'account.total' | translate:{ total: (kbsTotal | async) || 0, public: (kbsPublic | async) || 0}"></span>
       </div>
+      <div class="container">
+        <h2>{{ 'account.total_queries' | translate }}</h2>
+        <div class="total-queries">{{ ((totalQueries | async) || '').toLocaleString() }}</div>
```

Comment:
Why not using Angular's builtin pipe for formatting numbers instead? They are listed in here: https://angular.io/api/common#pipes, I think the one you're looking for is DecimalPipe: https://angular.io/api/common/DecimalPipe

Reply:
looks good, I'll fix it the next PR